README.md

```diff
@@ -1,26 +1,60 @@
-- **render_context** - platform-specific rendering context. Should be implemented on the application side.
+1. Copy [**OEP-module**](https://github.com/Banuba/OEP-module) to your project, or make it as a [**submodule**](https://git-scm.com/book/en/v2/Git-Tools-Submodules);
+2. Add to your **CMakeLists.txt** the subfolder that contains the **OEP-module**
+3. Write an implementation of the following interfaces: **effect_player** and **render_context**
+4. Now you can use the OEP-module in your code
 
-## Structure
+This module is configurable, see [**CMakeLists.txt**](./CMakeLists.txt) for details.
+If the logic of the OEP module does not allow you to do what you want, then you can implement interfaces that should have logic that is different from the current implementation. For example by default the OEP module works with OpenGL, if you want to use other rendering API then you have to write your own implementation [**offscreen render target**](./interfaces/offscreen_render_target.hpp) like this [**example**](https://github.com/Banuba/OEP-macos).
 
-- **offscreen_effect_player** - is a wrapper for effect_player. It allows you to use your own implementation for offscreen_render_target
-- **offscreen_render_target** - is an implementation option for the offscreen_render_target interface. Allows to prepare gl framebuffers and textures for receiving a frame from gpu, receive bytes of the processed frame from the gpu and pass them to the cpu, as well as, if necessary, set the orientation for the received frame. This implementation uses GLFW to work with gl context
-- **interfaces** - offscreen effect player interfaces
-- **opengl** - OpenGL utilities
+## List of examples using the OEP-module
 
-## Note
+All of the projects listed below have an implementation on the [**CPP-API**](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview) or on the **C-API**. [**You can read more details here**](https://docs.banuba.com/face-ar-sdk-v1/)
+- [OEP-desktop](https://github.com/Banuba/OEP-desktop) Windows and MacOS platforms with using OpenGL and Banuba SDK cpp-API
+- [OEP-desktop-c-api](https://github.com/Banuba/OEP-desktop-c-api) Windows and MacOS platforms with using OpenGL and Banuba SDK c-API
+- [quickstart-android-c-api](https://github.com/Banuba/quickstart-android-c-api) Android platform with using OpenGL and Banuba SDK c-API
+- [agora-plugin-filters-android](https://github.com/Banuba/agora-plugin-filters-android) Android platform with using OpenGL and Banuba SDK cpp-API
+- [OEP-macos](https://github.com/Banuba/OEP-macos) MacOS platform with using Metal and Banuba SDK cpp-API
+- [OEP_ios_c_api](https://github.com/sdk-banuba/OEP_ios_c_api) IOS platform with using OpenGL and Banuba SDK c-API
 
-To integrate the Offscreen effect player into your application, you need to copy the submodule folder and implement interfaces in which there must be logic different from the current implementation.
+## Backward compatibility
```

Comment:
I think this section should be removed since it is non-informative and introduces only confusion. As mentioned above, the module is independent of SDK, so backward compatibility is unrelated to the module.